datahub-web-react/src/app/ingest/source/builder/RecipeBuilder.tsx

```diff
@@ -65,16 +73,57 @@ function RecipeBuilder(props: Props) {
         }
     }
 
+    let link: React.ReactNode;
+    if (type === 'looker') {
+        link = (
+            <a href={LOOKER_DOC_LINK} target="_blank" rel="noopener noreferrer">
+                DataHub looker module
+            </a>
+        );
+    } else if (type === 'lookml') {
+        link = (
+            <a href={LOOKML_DOC_LINK} target="_blank" rel="noopener noreferrer">
+                DataHub lookml module
+            </a>
+        );
+    }
+
     return (
         <div>
-            <ButtonsWrapper>
-                <StyledButton type="text" isSelected={isViewingForm} onClick={() => switchViews(true)}>
-                    <FormOutlined /> Form
-                </StyledButton>
-                <StyledButton type="text" isSelected={!isViewingForm} onClick={() => switchViews(false)}>
-                    <CodeOutlined /> YAML
-                </StyledButton>
-            </ButtonsWrapper>
+            {(type === 'looker' || type === 'lookml') && (
+                <Alert
+                    type="warning"
+                    banner
+                    message={
+                        <>
+                            <big>
+                                <i>
+                                    <b>You must acknowledge this message to proceed!</b>
+                                </i>
+                            </big>
+                            <br />
+                            <br />
+                            To get complete Looker metadata integration (including Looker views and lineage to the
+                            underlying warehouse tables), you must <b>also</b> use the {link}.
+                        </>
+                    }
+                />
+            )}
```

Comment:
I think you can move this as well as the logic above regarding `link` into a new file and component called `LookerWarning` or something to keep this component a little cleaner.

Reply:
Created and fixed.